typeset/converter.js

```diff
@@ -131,37 +131,44 @@ const createMapOfMathMLElements = async (log, inputPath, cssPath, outputPath, ou
     return [...mathMap.entries()]
   }, PROGRESS_TIME)
 
-  const [convertedMathML/*: Map<string, {svg || html}> */, allUniqueCss] = await mjnodeConverter.convertMathML(log, new Map(mathEntries), outputFormat)
-
-  log.info(`Inserting converted math elements...`)
-  await page.evaluate((convertedMathMLEntries, PROGRESS_TIME) => {
-    const total = convertedMathMLEntries.length
-    let prevTime = Date.now()
-    let index = 0
-    for (const [id, xml] of convertedMathMLEntries) {
-      const mathHTML = xml
-      const mathNode = document.getElementById(id)
-      if (!mathNode) {
-        throw new Error(`BUG: Could not find element with id="${id}"`)
-      }
-      try {
-        mathNode.outerHTML = mathHTML
-      } catch (err) {
-        console.error(`Problem inserting id="${id}" back into the document (might not be valid XML)`)
-        console.error(mathHTML)
-        mathNode.outerHTML = mathHTML
-      }
+  const BATCH_SIZE = 3000
```

Comment:
Can we allow user to set custom BATCH_SIZE using some kind of flag?

Reply:
I'm sure that's possible.

Reply:
I've added `--batch-size` or `-b`.  I'm using a regexp to check if it's a valid integer.  Don't know if that's normal :sweat_smile: